public/components/trace_analytics/components/services/services_content.tsx

```diff
@@ -105,9 +106,8 @@ export function ServicesContent(props: ServicesProps) {
     props.dataSourceMDSId,
   ]);
 
-  const refresh = async (currService?: string, overrideQuery?: string) => {
+  const refresh = (currService?: string, overrideQuery?: string) => {
```

Comment:
Double checking if removing async is correct here

Reply:
Don't need `async` as we removed the `awaits`.